library/src/main/java/com/billing/google/GoogleBillingService2.kt

```diff
@@ -114,10 +117,8 @@ class GoogleBillingService2(context: Context, private val inAppSkuKeys: List<Str
             }
             BillingClient.BillingResponseCode.USER_CANCELED ->
                 log("onPurchasesUpdated: User canceled the purchase")
-            BillingClient.BillingResponseCode.ITEM_ALREADY_OWNED -> {
-                log("onPurchasesUpdated: The user already owns this item $purchases")
-                processPurchases(purchases, true)
```

Comment:
Is there any harm in also processing this?

I often see that it seems the Play Store cache does not have the purchase, but then the user tries to buy it and then the back end server says "it is already purchased" but then it is still not restored.